### ISSUE TITLE
Unity Sound Menu

### ISSUE THREAD

**Issue by panchal386** [snwh/paper-gtk-theme#112]
Ubuntu 14.04.2
Unity 7.2.4

![menu_001](https://cloud.githubusercontent.com/assets/6555461/6658274/ed0e4f4c-cb89-11e4-9905-38ad5c3cd646.png)
No Controls in Paper-gtk Theme

![menu_002](https://cloud.githubusercontent.com/assets/6555461/6658276/f3ddc992-cb89-11e4-92b5-8c9173c010d3.png)
Faience-gtk have controls

**Comment by nahtnam** [comment]
+1

**Comment by panchal386** [comment]
![menu_001](https://cloud.githubusercontent.com/assets/6555461/6841657/a34289fa-d3b3-11e4-9619-a00b1d0b4742.png)

**Comment by nahtnam** [comment]
@snwh any update on when this will be fixed?

Its using unity.

**Comment by julianinsua** [comment]
same issue, Would be great if it gets solved. Buttons are working in my case. But I've got tu guess where they were.

**Comment by nahtnam** [comment]
Yep. They are there but invisible.

**Comment by GiovanniCardamone** [comment]
+1

**Comment by ajmcsurley** [comment]
Looking for a solution to this as well.

**Comment by snwh** [comment]
Still an issue?

**Comment by ajmcsurley** [comment]
Yes, sadly. :( Updated to latest version I saw released but no luck.

**Comment by Loae** [comment]
i still get this bug....
